Makefile

```diff
@@ -14,7 +14,8 @@ include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machine
 # $2 - Dockerfile path
 # $3 - context directory for image build
 # It will generate target "image-$(1)" for builing the image an binding it as a prerequisite to target "images".
-$(call build-image,origin-$(GO_PACKAGE),./images/sdn/Dockerfile,.)
+$(call build-image,origin-sdn,./images/sdn/Dockerfile,.)
```

Comment:
reflect the real names here pls so we avoid confusion

Reply:
done